packages/mobile/src/analytics/Events.tsx

```diff
@@ -105,6 +109,12 @@ export enum OnboardingEvents {
   pin_failed_to_set = 'pin_failed_to_set',
   pin_never_set = 'pin_never_set',
 
+  biometric_verification_start = 'biometric_verification_start',
+  biometric_verification_cancel = 'biometric_verification_cancel',
+  biometric_verification_approve = 'biometric_verification_approve',
+  biometric_verification_complete = 'biometric_verification_complete',
+  biometric_verification_error = 'biometric_verification_error',
```

Comment:
nit: I'm wondering if we should name these differently.
```suggestion
  // User is prompted to enable biometry
  biometry_optin_start = 'biometriy_optin_start',
  biometriy_optin_cancel = 'biometriy_optin_cancel',
  biometriy_optin_approve = 'biometriy_optin_approve',
  biometriy_optin_complete = 'biometriy_optin_complete',
  biometriy_optin_error = 'biometriy_optin_error',
```

or maybe the `enable_biometry_` suffix?

Reply:
🙈 i had taken the list of events Gowtham gave me as gospel, but nice to know we can decide on these! i've let him know about the updates, i think `biometry_opt_in_*` makes sense the most (especially below with the settings enable event, `settings_enable_biometry_enable` didn't look nice